.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -37,6 +37,7 @@ Add as many details as possible to help others reproduce the issue and test the
 
 - [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
 - [ ] Covered with tests (or have a good reason not to test in description ☝️)
+- [ ] All strings are ready for i18n (internationalization)
```

Comment:
This is an extra check added to the PR description for future PRs

Reply:
What do you think about linking to the [package's README](https://github.com/WordPress/gutenberg/tree/trunk/packages/i18n)?

I was planning to P2 this change after merging it, but I believe it should be before as you suggested. Will do it 👍

Reply:
Created a P2 to ask for feedback. The summary is here paJDYF-cz6-p2#comment-23249

I will remove the checkbox for now and move forward with the changes.

Reply:
Thanks for starting a wider discussion. Marking this thread resolved while we discuss a solution on P2.